src/tapis-hooks/src/context/TapisContext.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Authenticator } from '@tapis/tapis-typescript';
+
+export type TapisContextType = {
+  accessToken: Authenticator.NewAccessTokenResponse,
```

Comment:
I think that the source of truth for the token should be a query with the fetcher `() => Cookies.getCookie('tapis-token')`. It lets us avoid the inevitable mismatch between the user's cookie (which could expire and be undefined) and the token stored in context. Because of how react-query handles focus, doing it this way will automatically boot the user to an unauthorized view instead of showing them a bunch of broken listings if they return after an expired session.

Reply:
Deferring resolution of this until later